spec/excon/rails_spec.rb

```diff
@@ -34,6 +34,14 @@ module Excon
         .to match(%r{Excon Request \(\d+\.\d+ms\)  GET http://example\.com/})
     end
 
+    it 'logs query params to logger' do
+      @logger.level = Logger::DEBUG
+      Rails::Railtie.run_initializers
+      Excon.get('http://example.com?foo=bar&baz=bar')
+      expect(@logger.logged(:debug)[0])
+        .to match(%r{Excon Request \(\d+\.\d+ms\)  GET http://example\.com/\?foo=bar&baz=bar})
```

Comment:
Line is too long. [94/80]<br>Place the . on the previous line, together with the method call receiver.